pkg/registries/adapters/adapter.go

```diff
@@ -130,3 +142,45 @@ func imageToSpec(log *logging.Logger, req *http.Request, apbtag string) (*apb.Sp
 
 	return spec, nil
 }
+
+func isCompatibleVersion(specVersion string, minVersion string, maxVersion string) bool {
+	specMajorVersion, err := strconv.Atoi(strings.Split(specVersion, ".")[0])
+	if err != nil {
+		return false
+	}
+	specMinorVersion, err := strconv.Atoi(strings.Split(specVersion, ".")[1])
```

Comment:
This could panic when I have set the version to `1` only.

Reply:
+1 another good catch